Clients/src/domain/types/AnnexCategoryISO.ts

```
@@ -0,0 +1,19 @@
+import { Status } from "./Status";
+
+export type AnnexCategoryISO = {
+  id?: number; // auto generated by database
+  is_applicable: boolean;
+  justification_for_exclusion: string;
+  implementation_description: string;
+  evidence_links: Object[];
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)